app/hotels/src/singleHotel/hotelInformation/description/Facilities.js

```diff
@@ -78,7 +78,7 @@ export class Facilities extends React.Component<Props, State> {
             facility && (
               <AdaptableBadge
                 key={facility.id}
-                text={facility.name || ''}
+                translation={<Translation passThrough={facility.name || ''} />}
```

Comment:
I think `Translation` is clever about this and can handle IDX without `|| ''`... 🙂